frappe/utils/user.py

```diff
@@ -157,8 +157,11 @@ def build_permissions(self):
 				self.can_read.remove(dt)
 
 		if "System Manager" in self.get_roles():
-			self.can_import = list(filter(lambda d: d in self.can_create,
-				frappe.db.sql_list("""select name from `tabDocType` where allow_import = 1""")))
+			docs = [x['name'] for x in frappe.get_all("DocType", "name")]
+			frappe.clear_cache()
```

Comment:
why? if you are getting cached meta then either use `frappe.get_meta(dt, cached=False)` or clear cache of just updated doctype eg. `frappe.cache().hdel('form_meta', 'User')` after updating from Customize Form.

Reply:
was unaware, alright